tgui/packages/tgui/interfaces/SecurityRecords.js

```diff
@@ -0,0 +1,435 @@
+        onClick={() => act('delete_security_all')}
+        mb="0.5rem"
+      /><br />
+      <Button.Confirm
+        icon="trash"
+        content="Delete All Cell Logs"
+        onClick={() => act('delete_cell_logs')}
+      />
+    </Box>
+  );
+};
+
+const SecurityRecordsPageView = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    isPrinting,
+    general,
+    security,
+  } = data;
+  if (!general || !general.fields) {
+    return (
+      <Box color="bad">
+        General records lost!
+      </Box>
+    );
+  }
+  return (
+    <Fragment>
+      <Section
+        title="General Data"
+        level={2}
+        mt="-6px"
+        buttons={
+          <Fragment>
+            <Button
+              icon={isPrinting ? 'spinner' : 'print'}
+              disabled={isPrinting}
+              iconSpin={!!isPrinting}
+              content="Print Record"
+              onClick={() => act('print_record')}
+            />
+            <Button.Confirm
+              icon="trash"
+              tooltip={"WARNING: This will also delete the Security "
+              + "and Medical records associated to this crew member!"}
+              tooltipPosition="bottom-left"
+              content="Delete Record"
+              onClick={() => act('delete_general')}
+            />
+          </Fragment>
+        }>
+        <SecurityRecordsViewGeneral />
+      </Section>
+      <Section
+        title="Security Data"
+        level={2}
+        mt="-12px"
+        buttons={
+          <Button.Confirm
+            icon="trash"
+            disabled={security.empty}
+            content="Delete Record"
+            onClick={() => act('delete_security')}
+          />
+        }>
+        <SecurityRecordsViewSecurity />
+      </Section>
+    </Fragment>
+  );
+};
+
+const SecurityRecordsViewGeneral = (_properties, context) => {
+  const { data } = useBackend(context);
+  const {
+    general,
+  } = data;
+  if (!general || !general.fields) {
+    return (
+      <Box color="bad">
+        General records lost!
+      </Box>
+    );
+  }
+  return (
+    <Fragment>
+      <Box float="left">
+        <LabeledList>
+          {general.fields.map((field, i) => (
+            <LabeledList.Item key={i} label={field.field}>
+              {decodeHtmlEntities('' + field.value)}
+              {!!field.edit && (
+                <Button
+                  icon="pen"
+                  ml="0.5rem"
+                  onClick={() => doEdit(context, field)}
+                />
+              )}
+            </LabeledList.Item>
+          ))}
+        </LabeledList>
+      </Box>
+      <Box position="absolute" right="0" textAlign="right">
+        {!!general.has_photos && (
+          general.photos.map((p, i) => (
+            <Box
+              key={i}
+              display="inline-block"
+              textAlign="center"
+              color="label">
+              <img
+                src={p.substr(1, p.length - 1)}
```

Comment:
Whats the reasoning for this? Are images wrapped in 2 sets of quotation marks?

Reply:
Yes. I'll remove them and see if they still work